src/Writer/Hive.php

```diff
@@ -135,13 +135,23 @@ public function create(array $table): void
         // Columns are ordered according CSV header, see Application::reorderColumns
         $columns = array_filter($table['items'], fn(array $item) => strtolower($item['type']) !== 'ignore');
         $columnsDefs = array_map(fn($column) => $this->createColumnDef($column), $columns);
-        $this->db->query(
-            'CREATE %sql TABLE %n (%sql) %sql',
-            !empty($table['temporary']) ? 'TEMPORARY' : '',
-            $table['dbName'],
-            implode(', ', $columnsDefs),
-            implode(' ', $tableSpec),
-        );
+        try {
+            $this->db->query(
+                'CREATE %sql TABLE %n (%sql) %sql',
+                !empty($table['temporary']) ? 'TEMPORARY' : '',
+                $table['dbName'],
+                implode(', ', $columnsDefs),
+                implode(' ', $tableSpec),
+            );
+        } catch (Dibi\DriverException $e) {
+            // HiveDB has some limitations, eg. table name cannot contains special characters (not even "-")
+            // Let the user know the error message
+            throw new UserException(sprintf(
```

Comment:
Pri testovani UI som zistil, ze sa nazov tabulky v Hive DB nemoze obsahovat specialne znaky, a dokonca ani pomlcku `-` (co sa dost bezne pouziva). Preto som to upravil tak, ze ak neprejde `CREATE TABLE`, tak sa nevyhodi `ApplicationException` ako doreaz ale `UserException`, aby uzivatel vedel aky je problem (napr. pomlcka v nazve tabulky).